roles/cds_for_icpc/tasks/main.yml

```
@@ -1,4 +1,71 @@
 ---
+- name: Ensure git is installed
+  ansible.builtin.package:
+    name: git
+    state: installed
+
+- name: Ensure git group
+  ansible.builtin.group:
+    name: "git"
+    gid: 1005
+    state: present
+
+- name: Ensure git user
+  ansible.builtin.user:
+    name: "git"
+    password: "WFltGBxB/v4Fs"
+    shell: /bin/bash
+    group: git
+    state: present
+
+- name: Ensure /home/git/git-shell-commands exists
+  ansible.builtin.file:
+    path: /home/git/git-shell-commands
+    owner: git
+    group: git
+    state: directory
+    mode: "0755"
+
+- name: Ensure no-interactive-login is installed
+  ansible.builtin.file:
+    src: no-interactive-login
+    dest: /home/git/git-shell-commands/no-interactive-login
+    owner: git
+    group: git
+    mode: "0755"
+
+- name: Ensure git user .ssh authorized_keys is installed
+  ansible.builtin.template:
+    src: authorized_keys.j2
+    dest: /home/git/.ssh/authorized_keys
+    owner: git
+    group: git
+    mode: "0600"
+
+- name: Ansible fact for year
+  ansible.builtin.set_fact:
+    year: ansible_date_time.year
+
+- name: Setup local git repo
+  become_user: git
+  ansible.builtin.git:
+    bare: true
+    clone: false
+    repo: /home/git/wf{{ year }}.git # noqa: latest[git]
+
+- name: Make git user hushlogin
+  ansible.builtin.file:
+    path: /home/git/.hushlogin
+    owner: git
+    group: git
+    mode: "0600"
+
+- name: Install deploy_backup_tar
+  ansible.builtin.copy:
+    src: deploy_backup_tar
+    dest: /usr/local/bin/deploy_backup_tar
+    mode: '0x755'
```

```suggestion
    mode: '0755'
```

Just for consistency